<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblTrans">
      <property name="text">
       <string>NO TRANSAKSI</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editTrans"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblKdZakat">
      <property name="text">
       <string>KODE ZAKAT</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblKdMuztahik">
      <property name="text">
       <string>KODE MUSTAHIK</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblJumlah">
      <property name="text">
       <string>JUMLAH</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editJumlah"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblBentuk">
      <property name="text">
       <string>BENTUK</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmbBentuk">
      <item>
       <property name="text">
        <string>Zakat Fitrah</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zakat Mal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zakat Emas dan Perak</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zakat Perdagangan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zakat Pertanian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zakat Peternakan</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblTglKel">
      <property name="text">
       <string>TANGGAL KELUAR</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="editTglKel"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblIdAdmin">
      <property name="text">
       <string>ID ADMIN</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lblKet">
      <property name="text">
       <string>KETERANGAN</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTextEdit" name="txtKet"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cmbKdZakat"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cmbKdMustahik"/>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="cmbIdAdmin"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>250</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>250</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblZakatKeluar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>601</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Kode Zakat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kode Mustahik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>No Transaksi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bentuk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal Masuk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Admin</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Keterangan</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CARI DATA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>591</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
